examples/acs2/maze/acs2_in_maze.py

```diff
@@ -21,29 +22,36 @@ def maze_metrics(population, environment):
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
```

Comment:
Please use Click library - https://click.palletsprojects.com/en/7.x/

Reply:
Why is it better than argparse?

Reply:
There is a whole section "Why Click"

Reply:
Argparse does its job. Since I already wrote this and it works, I suggest that we keep this for now and leave Click for the future as a nice-to-have.

Reply:
Ok